scripts/create_completeness_report.py

```diff
@@ -73,7 +75,7 @@ def match_datsets(record, files, geoextent_all):
                 and (record[FLD_THEME] == '*' or theme == record[FLD_THEME]) \
                 and (record[FLD_SOURCE] == '*' or record[FLD_SOURCE] == source)\
                 and geometry_type == record[FLD_TYPE]:
-            #print (dataset_name & dateTime)
+            # print (dataset_name & dateTime)
```

Comment:
let's avoid having commented code.

Reply:
I tried to add explanation. I'll correct there.